app/src/test/java/org/simple/clinic/teleconsultlog/drugduration/DrugDurationInitTest.kt

```diff
@@ -18,7 +18,7 @@ class DrugDurationInitTest {
         .whenInit(model)
         .then(assertThatFirst(
             hasModel(model),
-            hasEffects(SetDrugDuration(duration) as DrugDurationEffect)
+            hasNoEffects()
```

Comment:
With this, the name of the test is out of sync with the assertion. Please rename the test (or even delete it)?

Reply:
Oh yes, completely forgot. Removing the test